proc_dash/utility.py

```diff
@@ -95,6 +104,12 @@ def count_unique_subjects(data: pd.DataFrame) -> int:
     )
 
 
+def count_unique_records(data: pd.DataFrame) -> int:
```

Comment:
is the idea here that you want to get the number of unique combinations of `participant_id` and `session`?

Reply:
Yes! It was actually Nikhil who suggested I call these participant-session rows "records" (I think borrowed from clinical database terminology), which I like. Docstring added for clarity.